server/data/mappers/conditionDtoMapper.ts

```diff
@@ -0,0 +1,19 @@
+import type { ConditionDto, ConditionsList } from 'dto'
+import type { ConditionListResponse, ConditionResponse } from 'supportAdditionalNeedsApiClient'
+import toReferenceAndAuditable from './referencedAndAuditableMapper'
+
+const toConditionsList = (conditionListResponse: ConditionListResponse, prisonNumber: string): ConditionsList => ({
```

Comment:
New mapper for the condition stuff - mapping a `ConditionListResponse` (API response type) into a `ConditionsList` (DTO type)